lib/generators/spotlight/install_generator.rb

```diff
@@ -10,6 +10,25 @@ class Install < Rails::Generators::Base
     class_option :solr_update_class, type: :string, default: 'Spotlight::SolrDocument::AtomicUpdates'
     class_option :mailer_default_url_host, type: :string, default: '' # e.g. localhost:3000
 
+    # we're not (yet) using webpacker, so we need to re-add sprockets functionality
+    def add_js
+      return unless ENV['RAILS_VERSION'].to_i == 6
```

Comment:
We may want to use `Rails.version` here instead of `ENV['RAILS_VERSION']`.

At the point of executing a generator we should have a rails context and know the version that is installed. I believe typical users won't be passing the `RAILS_VERSION` env var when installing spotlight and will use whatever the most recent version of rails they have is.

Reply:
fixed with https://github.com/projectblacklight/spotlight/pull/2371/commits/379001852c36b95188ed44b5a2ecf5c16ce97aec